ChangeLog.md

```diff
@@ -23,6 +23,10 @@ v1.38.33: 05/23/2019
 
  - Add `emscripten_return_address` which implements the functionality of
    gcc/clang's `__builtin_return_address`. (#8617)
+ - Add support for [undefined behavior sanitizer](https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html).
```

Comment:
Should this code in the current trunk section above?

Reply:
Thanks for catching this. Git merge messed this up.